test/unit/healthcheck_test.exs

```diff
@@ -0,0 +1,94 @@
+defmodule MongoosePushWeb.HealthcheckTest do
+  alias MongoosePush.Support.API
+  use ExUnit.Case, async: false
+  import Mox
+
+  setup :verify_on_exit!
+
+  setup do
+    TestHelper.reload_app()
+  end
+
+  test "Successful connection to services" do
+    {_, _, body} = API.get("/healthcheck")
```

Comment:
I think it'd worth to check the status of the response as well. If I understand it correctly this is `200`.

Reply:
>     3. Open question. Is status 200 a good fit for a situation when all connections are dead? Software like load balancers decides if a service is a health based on the HTTP status code.
I agree, I put 503 status response in that case. However, I'm not completely sure if this "service unavailable" would be about APNS and FCM or the MPush itself

Reply:
Maybe this can be used as a guideline: https://tools.ietf.org/id/draft-inadarei-api-health-check-01.html :)

Reply:
That's good reading. Thanks for that @rslota. @kmakiela could you take this document into account?